network/wsNetwork_test.go

```diff
@@ -1302,9 +1384,9 @@ func TestForceMessageRelaying(t *testing.T) {
 
 	// send 5 messages from both netB and netC to netA
 	for i := 0; i < 5; i++ {
-		err := netB.Relay(context.Background(), protocol.TxnTag, []byte{1, 2, 3}, true, nil)
+		err := netB.Relay(context.Background(), protocol.TxnTag, []byte{byte(i), 2, 3}, true, nil)
```

Comment:
is that because of the filtering you've added ( but excluded from this PR ) ? if so, than this change shouldn't be included here either.